src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/win_proxy.py

```diff
@@ -3,31 +3,67 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
-from ctypes import (
-    byref,
-    c_ulong,
-    create_unicode_buffer,
-    POINTER,
-    sizeof,
-    Structure,
-    Union,
-    windll)
-from ctypes.wintypes import (
-    BOOL,
-    DWORD,
-    FILETIME,
-    LPVOID,
-    WCHAR)
+from ctypes import (POINTER, Structure, Union, byref, c_ulong,
+                    create_unicode_buffer, sizeof, windll)
+from ctypes.wintypes import BOOL, DWORD, FILETIME, LPVOID, WCHAR
+
+from azure.cli.command_modules.acs.proxy import Proxy
 
 LPWSTR = POINTER(WCHAR)
 HINTERNET = LPVOID
 
-INTERNET_PER_CONN_PROXY_SERVER = 2
-INTERNET_OPTION_REFRESH = 37
-INTERNET_OPTION_SETTINGS_CHANGED = 39
-INTERNET_OPTION_PER_CONNECTION_OPTION = 75
-INTERNET_PER_CONN_PROXY_BYPASS = 3
-INTERNET_PER_CONN_FLAGS = 1
+class WinProxy(Proxy):
+    INTERNET_PER_CONN_PROXY_SERVER = 2
+    INTERNET_OPTION_REFRESH = 37
+    INTERNET_OPTION_SETTINGS_CHANGED = 39
+    INTERNET_OPTION_PER_CONNECTION_OPTION = 75
+    INTERNET_PER_CONN_PROXY_BYPASS = 3
+    INTERNET_PER_CONN_FLAGS = 1
+
+    def __init__(self):
+        super(WinProxy, self).__init__()
+
+    def set_http_proxy(self, host, port):
+        """
+        Sets the HTTP proxy on Windows
+        """
+        setting = create_unicode_buffer(host+':'+str(port))
+        self.__set_internet_options(setting)
+
+    def disable_http_proxy(self):
+        """
+        Disables the HTTP proxy
+        """
+        self.__set_internet_options(None, False)
+
+    def __set_internet_options(self, setting, on=True):
```

Comment:
It should be a single underscore for private members.

This link describes the convention.
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables
"Use one leading underscore only for non-public methods and instance variables. "